react/components/organisms/listing/listing.js

```diff
@@ -279,14 +435,38 @@ const styles = StyleSheet.create({
         flex: 1,
         backgroundColor: "#f6f7f9"
     },
-    search: {
+    searchingHeader: {
+        flex: 1,
+        marginBottom: -2,

```

Comment:
Isn't there a way of removing from somewhere else instead of using negative margins? Just asking since we normally don't go with this approach, but if it is the only one